tools/src/versioning/android/index.ts

```diff
@@ -432,14 +439,56 @@ async function copyExpoModulesAsync(version: string) {
       pkg.isIncludedInExpoClientOnPlatform('android') &&
       pkg.isVersionableOnPlatform('android')
     ) {
-      await spawnAsync(
-        './android-copy-expo-module.sh',
-        [pkg.packageName, version, path.join(pkg.path, pkg.androidSubdirectory)],
-        {
-          shell: true,
-          cwd: SCRIPT_DIR,
+      const abiVersion = `abi${version.replace(/\./g, '_')}`;
+      const targetDirectory = path.join(ANDROID_DIR, `versioned-abis/expoview-${abiVersion}`);
+      const sourceDirectory = path.join(pkg.path, pkg.androidSubdirectory);
+      const transforms = expoModulesTransforms(pkg.packageName, abiVersion);
+
+      const files = await searchFilesAsync(sourceDirectory, [
+        './src/main/java/**',
+        './src/main/kotlin/**',
+        './src/main/AndroidManifest.xml',
+      ]);
+
+      for (const javaPkg of packagesToKeep) {
+        const javaPkgWithSlash = javaPkg.replace(/\./g, '/');
+        const pathFromPackage = `./src/main/{java,kotlin}/${javaPkgWithSlash}{/**,*}`;
```

Comment:
I'm not sure what to do about `{/**,*}`, in my opinion, it should be `{/**,.java,.kt}`

With current behavior(or with bash script one) if we have `expo.modules.splashscreen.singletons.SplashScreen` in packages to keep it would match
- `expo/modules/splashscreen/singletons/SplashScreen.java`
- `expo/modules/splashscreen/singletons/SplashScreenStatusBar.java`

If we would go with my preferred option it would only match `SplashScreen.java`, so we would need to add `expo.modules.splashscreen.singletons.SplashScreenStatusBar` to the list.